topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+>
+> 1. {% tool [RDConf: Low-energy ligand conformer search](toolshed.g2.bx.psu.edu/repos/bgruening/rdconf/rdconf/2020.03.4+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `outfile` (output of **Add hydrogen atoms** {% icon tool %})
+>    - *"Maximum number of conformers to generate per molecule"*: `100`
+>
+>    > ### {% icon comment %} RDConf
+>    >
+>    > It is recommended to use the default settings, except for the number of conformers which should be changed to 100. As a rule of thumb, a threshold of 100 conformers appropriately represents the conformational flexibility of a compound with less than 10 rotatable bonds. The output SDF (structure data file) format encodes three-dimensional atomic coordinates of each conformer, separated by lines containing four dollar signs (*$$$$*).
+>    {: .comment}
+>
+{: .hands_on}
+
+# Pharmacophore alignment
+
+In this step, the ligand conformer dataset (SDF format) is converted on-the-fly to a pharmacophore dataset (PHAR format) and simultaneously aligned to the individual pharmacophores of the ePharmaLib dataset in a batch mode with Align-it ({% cite Taminau2008 %}). The pharmacophoric alignments and thus the predicted targets are ranked in terms of a scoring metric: `Tversky index` = [0,1]. The higher the Tversky index, the higher the likelihood of the predicted protein--ligand interaction. A cut-off of 0.5 is applied to prioritize the most promising targets.
+
+> ### {% icon hands_on %} Hands-on 7: Pharmacophore alignment
+>
+> 1. {% tool [Pharmacophore alignment](toolshed.g2.bx.psu.edu/repos/bgruening/align_it/ctb_alignit/1.0.4+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Defines the database of molecules that will be used to screen"*: `outfile` (output of **RDConf: Low-energy ligand conformer search** {% icon tool %})
+>    - {% icon param-file %} *"Reference molecule"*: `list_output_generic` (output of **Split file** {% icon tool %})
+>    - *"No normal information is included during the alignment"*: `Yes`
+>    - *"Disable the use of hybrid pharmacophore points"*: `Yes`
+>    - *"Only structures with a score larger than this cutoff will be written to the files"*: `0.5`
+>    - *"Maximum number of best scoring structures to write to the files"*: `1`
+>    - *"This option defines the used scoring scheme"*: `TVERSKY_REF`
+>
+{: .hands_on}
+
+This step takes about 45 minutes to an hour to complete.
+
+> ### {% icon tip %} Tip: Problems using the Align-it tool?
+> Due to the large number of files contained in the ePharmaLib dataset collection, users occasionally encounter technical issues with this particular Align-it job. Sometimes the job takes too long to successfully complete (more than the estimated 1 hour duration) depending on the available Galaxy computing resources. And in other cases the job completes with errors; if this happens, delete the errored outputs and rerun the job.
+{: .tip}
```

Comment:
I guess you can remove this tip after using a smaller subset of the dataset. For a tutorial, it would make sense to only include pharmacophores in the provided subset for which you are confident the tool will succeed. :)

Reply:
I replaced the dataset ePharmaLib_human (5010 pharmacophores) with ePharmaLib_plasmodium (138 pharmacophores), and deleted the tip as suggested.

Reply:
Cool, if there is a smaller 'real dataset' that is even better!